packages/compass/src/app/styles/index.less

```diff
@@ -12,7 +12,6 @@
 @import './deprecated-bootstrap-styles/grid.less';
 
 // Dependencies
-@import './react-select.less';
```

Comment:
Can we also remove this file now? Or are we going to clean up all of the files later on?